paddy/writer.py

```diff
@@ -95,11 +95,13 @@ def box(g_data, box_type):
     """
     plot_data = []
     x_val = []
+    c = 0
     for i in g_data:
-        temp = g_data[i]
-        x_val.append(int(i))
+        temp = g_data['{0}'.format(c)]
+        x_val.append(c)
         data = temp
         plot_data.append(data)
+        c += 1
```

Comment:
Can you write a comment about what this does? Why aren't you using a `range` for `x_val`. Why are you using a dictionary of integer strings?